Makefile

```diff
@@ -4,7 +4,22 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# The time limit in seconds for each test
+TIMEOUT ?= 60
+
 default: checkcommits
 
 checkcommits:
 	make -C cmd/checkcommits
+
+ginkgo:
+	ln -sf . vendor/src
+	GOPATH=$(PWD)/vendor go build ./vendor/github.com/onsi/ginkgo/ginkgo
+	unlink vendor/src
+
+integration: ginkgo
+ifeq ($(RUNTIME),)
```

Comment:
Nice! I personally tend to put checks like this above all the rules since it's starts to get confusing and you end up with duplication when you need to check multiple variables. However, not required here as you are only checking a single variable.

There is one gotcha with my approach though btw: you must remember **not** to use a tab when indenting `$(error)`:

```
# XXX: The indent on error line must not use tabs since if the check is successful (RUNTIME not set)
# XXX: make will get unhappy about the tab as it will expect a rule.
ifeq ($(RUNTIME),)
    $(error "ERROR: RUNTIME is not set")
endif

    :

# XXX: of course the indent for ginkgo does have a tab here ;)
integration: ginkgo
    ./ginkgo -v -focus "${FOCUS}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
```